### DISCUSSION TITLE
Motion Planning in Pybullet

### DISCUSSION THREAD

**Discussion by kaku-KRS** [bulletphysics/bullet3#3776] [General]
Hi

Is there a motion planning function integrated into pybullet, like Moveit! in ROS?

It's easy to use than Moveit Planning scene in simulation, but luck of a motion planning pipeline.

Something like OMPL-RRT with colligion detection to navigate a robot arm through the environment is needed.

Or any suggested library for this?

**Comment by seann999** [comment]
Bump

**Comment by kaku-KRS** [comment]
> Bump

Where
